src/app/routes/article/getInitialData/index.ts

```diff
@@ -19,6 +20,16 @@ type Props = {
   path: string;
   pageType: 'article' | 'cpsAsset';
   variant?: Variants;
+  toggles?: Toggles;
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const addDisclaimer = async (pageData: any, toggles: Toggles | undefined) => {
+  try {
+    return augmentWithDisclaimer(toggles, 0)(pageData);
```

Comment:
Could we make the second argument here a named const or define an [object parameter](https://github.com/bbc/simorgh/blob/1a8f8eb6bbb1a6e8b8e4342f177a323aee0a94ed/docs/Coding-Standards/Clean-Code.stories.mdx#L185-L220) in the function definition for `augmentWithDisclaimer` so it's more clear what this number does?

Reply:
Good thinking!